build/all/TF_VERSION

```diff
@@ -0,0 +1 @@
+0.12.29
```

Comment:
Shall we maintain the `TF_VERSION` on root level and read it from there (or symlink it, at least)? I guess it makes sense to allow providers to overwrite the used Terraform version, but so far we always used the same (and probably want to continue this journey?), so maintaining this in a single file instead of copying it n-times is easier for now.

Reply:
Yes, good suggestion. Will do the symlink then.